src/display/GraphicsHashtable.java

```diff
@@ -0,0 +1,42 @@
+package display;
+
+import enums.Graphics;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.nio.Buffer;
+import java.util.Calendar;
+import java.util.Hashtable;
+
+import static enums.Graphics.*;
+
+public class GraphicsHashtable {
+    public static  Hashtable<Graphics, BufferedImage> images = null;
+    //wyjatek obsluzylibysmy w superklasie map display albo project manager - tam tez byloby jedyne wywolanie metody
+    //tylko jak to zrobic zeby wywolala sie dokladnie raz? (Po namysle mozna sprawdzic czy images to null ale to chyba nieladne)
```

Comment:
Można zrobić prywatne pole i statyczną metodę `public static File getImage(Graphics graphics)`, w której sprawdzamy, czy to null i wciśniemy to Tabakowowi jako singletona. Opcjonalnie możesz poczytać czym jest Singleton z early initialization.